packages/google_maps_flutter/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -8,7 +8,8 @@
         android:name="io.flutter.app.FlutterApplication"
         android:label="google_maps_flutter_example"
         android:icon="@mipmap/ic_launcher">
-      <meta-data
+        <uses-library android:name="org.apache.http.legacy" android:required="false"/>
```

Comment:
We needed this due to a bug that existed. We no longer need this change. Could you please revert all changes to this file.